docker-compose/goloop2moonbeam/README.MD

```diff
@@ -14,22 +16,35 @@ This document will walk you through setting up a Docker environment to demonstra
 
 ## Setup steps:  
 
-For easy setup, we recommend to use a fresh ubuntu 21.04. Hence, run those commands to installe prerequire tools
+For easy setup, we recommend to use a fresh ubuntu 18.04. Hence, run those commands to installe prerequisite tools
 ```
 apt update
 apt install git make zip golang-go docker.io docker-compose --fix-missing -y
 sudo groupadd docker
 sudo usermod -aG docker $USER
 ```
 
+For Mac OS, we recommend to use homebrew for prerequisites
+```
+brew install git make go dokcer docker-compose
+```
+
+## Computer specs
+
+At least 8 CPUs, 16 RAM.
```

Comment:
@trmaphi I don't think it costs so much ram ?

Reply:
For safety threshold

Reply:
4g is enough I think so. Since, docker get all of ram the host will be slow

Reply:
<img width="820" alt="Screen Shot 2021-09-03 at 15 22 33" src="https://user-images.githubusercontent.com/2246996/131974554-6babe1ec-1f67-46f7-b6db-2cabeee4abc7.png">